.github/workflows/build_nix.yml

```
@@ -1,10 +1,16 @@
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
 on:
   push:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  schedule:
+    - cron: "* 12 1 * *"
```

actionlint (1.7.4)

scheduled job runs too frequently. it runs once per 60 seconds. the shortest interval is once every 5 minutes

(events)